plugins/importexport/csv/CSVImportExportPlugin.inc.php

```diff
@@ -82,177 +82,330 @@ function executeCLI($scriptName, &$args) {
 
 		$filename = array_shift($args);
 		$username = array_shift($args);
+		$basePath = dirname($filename);
 
 		if (!$filename || !$username) {
 			$this->usage($scriptName);
 			exit();
 		}
 
 		if (!file_exists($filename)) {
-			echo __('plugins.importexport.csv.fileDoesNotExist', array('filename' => $filename)) . "\n";
+			echo __('plugins.importexport.csv.fileDoesNotExist', ['filename' => $filename]) . "\n";
 			exit();
 		}
 
-		$data = file($filename);
+		$data = file($filename, FILE_IGNORE_NEW_LINES | FILE_SKIP_EMPTY_LINES);
+		str_getcsv(array_shift($data)); // Remove column headers in first row
 
 		if (is_array($data) && count($data) > 0) {
```

Comment:
You're checking here if `$data` is an array, but the `str_getcsv(array_shift($data))` above already assumes that it is. If you like, you can replace the above check for `file_exists` with the call to `file` and let it do the error checking:

```php
$data = file($filename, FILE_IGNORE_NEW_LINES | FILE_SKIP_EMPTY_LINES)
if ($data === false) {
	echo __('plugins.importexport.csv.fileDoesNotExist', array('filename' => $filename)) . "\n";
	echo __('plugins.importexport.csv.fileDoesNotExist', ['filename' => $filename]) . "\n";
	exit();
}

if (count($data > 0) {
	...etc...
```